feat(STONEINTG-966): ensure override snapshot is valid to be released (#799)
konflux-ci/integration-service: docs/snapshot-controller.md

  rerun_static_env                ---->    remove_rerun_label

  %%%%%%%%%%%%%%%%%%%%%%% Drawing EnsureOverrideSnapshotValid() function

  %% Node definitions
  ensure4(Process further if: Snapshot has override type label)

>   validate_override_valid{Check if override snapshot has <br>valid image digest and git source}

Reviewer: this should be phrased as a question I think
```suggestion
  validate_override_valid{Is the override snapshot <br>defined with valid components, <br>image digest, and git source?}
```

>   mark_snapshot_invalid(<b>Mark</b> overide snapshot as invalid)

Reviewer: nit:
```suggestion
  mark_snapshot_invalid(<b>Mark</b> override snapshot as invalid)
```
